components/authz-service/server/v2/project_update_manager_test.go

```diff
@@ -46,7 +49,25 @@ func loadWorkflowFromDisk(t *testing.T, name string) *workflowInstance {
 	} else {
 		defer fPayload.Close()
```

Comment:
You should get rid of all occurrences of `fPayload` which is no longer being used.